### PR TITLE
board-test: update to version 0.1.2

### PR DESCRIPTION
**Pull request by francois-berder-imgtec**
This connects to CreatorDev/openwrt#185

Signed-off-by: Francois Berder <francois.berder@imgtec.com>

### CODE REVIEW

File: board-test/Makefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=board-test
-PKG_VERSION:=0.1.1
-PKG_RELEASE:=3
+PKG_VERSION:=0.1.2
+PKG_RELEASE:=1
```
Comment (nikhil-zinjurde-imgtec) [inline board-test/Makefile]: this should be 4 i.e. incremented as the nth release of this package Makefile

PKG_RELEASE - The version of this package Makefile
https://wiki.openwrt.org/doc/devel/packages

Comment (francois-berder-imgtec) [inline board-test/Makefile]: Based upon discussion [here](https://github.com/openwrt/packages/issues/149), it seems that PKG_RELEASE must be reset to 1 whenever PKG_VERSION is changed.

Comment (nikhil-zinjurde-imgtec) [inline board-test/Makefile]: OK, that's for clarifying.  @abhijit-mahajani-imgtec FYI

Comment (abhijit-mahajani-imgtec) [inline board-test/Makefile]: Ok thanks @francois-berder-imgtec. and @nikhil-zinjurde-imgtec for the info.

**Pull request merged**